libs/coin-modules/coin-polkadot/src/network/node/nominations.ts

```diff
@@ -24,37 +23,45 @@ export const fetchNominations = async (address: string): Promise<SidecarNominati
       targets: [],
     };
   }
-
   const { targets, submittedIn } = nominationsOpt.unwrap();
 
-  const [exposures, stashes] = await Promise.all([
-    api.query.staking.erasStakers.multi<Exposure>(targets.map(target => [activeEra, target])),
-    api.derive.staking.stashes(),
-  ]);
+  const stashes = await api.derive.staking.stashes();
 
   const allStashes = stashes.map(stash => stash.toString());
-
+  const returnTargets = [];
+  for (const target of targets) {
+    let status: "active" | "inactive" | "waiting" | null = null;
+    let value = "0";
+    const exposure = await api.query.staking.erasStakersOverview(activeEra, target.toString());
```

Comment:
is there a erasStakers.multi equivalent ?

Reply:
I don't think so. We can't retrieve all the nominations once now. We should retrieve them page by page now.
The "erasStakers" is deprecated.
Refer to the change log of v14
https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame/staking/CHANGELOG.md#deprecated-1

Reply:
I don't think so.
Instead of retrieve all the nominations once, we need to fetch nominations page by page now.
Refer to 
https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame/staking/CHANGELOG.md#deprecated-1
Since v14, the "erasStakers" is deprecated. Instead, we should use erasStakersOverview + erasStakersPaged to get all the nominations.